src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const total = items.length;
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [itemsPerPage, setItemsPerPage] = useState<number>(5);
+
+  const handleChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setCurrentPage(1);
+    setItemsPerPage(+e.target.value);
+  };
```

Comment:
The `handleChange` function directly converts the event target value to a number using the unary plus (+). While this works, it might be more readable to use `parseInt` with a radix for clarity, e.g., `parseInt(e.target.value, 10)`.